src/lib/SCEditor.js

```diff
@@ -379,6 +380,56 @@ export default function SCEditor(original, userOptions) {
 	// Don't deep extend emoticons (fixes #565)
 	base.opts.emoticons = userOptions.emoticons || defaultOptions.emoticons;
 
+	if (!Array.isArray(options.allowedIframeUrls)) {
+		options.allowedIframeUrls = [];
+	}
+	options.allowedIframeUrls.push('https://www.youtube-nocookie.com/embed/');
+
+	// Create new instance of DOMPurify for each editor instance so can
+	// have different allowed iframe URLs
+	// eslint-disable-next-line new-cap
+	var domPurify = DOMPurify();
```

Comment:
Could you expose this via a method? That would enable us to add our own hooks.

Reply:
Ideally I'd like to not expose dompurify if possible as it would make it difficult to swap out if ever needed. Is there anything you'd need to add a hook for? If there is then it'll have to be exposed but I'd prefer to avoid it if possible.